teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -141,7 +269,214 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowCallQueueOptOutChange
+
+When set to True users affected by the policy will be allowed to change the call queue whether to allow agent opt-out. When set to False (the default value) users affected by the policy will not be allowed to change the call queue whether to allow agent opt-out.
```

Comment:
... allowed to change the call queue opt-out setting that allows agents to opt-out of receiving calls.
.... will not be allowed to change the call queue opt-out setting that allows agents to opt-out of receiving calls.